panopticum/admin.py

```diff
@@ -269,6 +270,12 @@ class RequirementSetAdmin(admin.ModelAdmin):
     model = RequirementSet
 
 
+class ComponentAdmin(admin.ModelAdmin):
+    search_fields = ['name']
+    list_display = ['name', 'type', 'data_privacy_class', 'category', 'subcategory', 'vendor']
```

Comment:
Why it is needed? Does it mean we hide 'description' and 'life_status'?

Reply:
before: